### DISCUSSION TITLE
請問字詞記憶的功能

### DISCUSSION THREAD

**Discussion by larryjclai** [vChewing/vChewing-macOS#139] [Q&A]
請問我要怎麼開啟字詞記憶的功能呢？
就是讓輸入法學習我打的一些常用字
舉個例子：台灣有一間餐廳叫「春水堂」，目前預設是「春"水塘"」，不論我打幾次春水堂他都還是不會記住
還是必須要自己編使用者詞庫來達成呢？

**Comment by ShikiSuen** [comment]
可以就地加詞。詳細操作步驟請點輸入法選單末端的「熱鍵指南」。
Shift+前後方向鍵在組字區內選中範圍，然後按照屆時的螢幕提示進行操作。

**Reply by larryjclai** [reply]
原來如此，幫了我一個大忙，太感謝了！
